apis/apps/v1alpha1/cluster_types.go

```diff
@@ -1216,6 +1221,27 @@ type ClusterComponentService struct {
 	PodService *bool `json:"podService,omitempty"`
 }
 
+type ClusterSystemAccount struct {
```

Comment:
ClusterSystemAccount -> ClusterComponentSystemAccount or  ComponentSystemAccount  seems more reasonable？

Reply:
Rename to ComponentSystemAccount.